zerver/lib/bugdown/__init__.py

```diff
@@ -693,7 +693,12 @@ def make_emoji(emoji_name, src, display_string):
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
-    elt.set("title", display_string)
+    if display_string[0] == ':':
+        # removes the colons at the ends and replaces space with underscore
+        elt.set("title", display_string[1:-1].replace("_", " "))
+    else:
+        # for unicode emojis
+        elt.set("title", display_string)
```

Comment:
I don't think this function should have an `if/else` structure; seems like you should change how it's called instead so that the string definitely either does or does not have the `:` (not having them would probably be cleaner).